client/hostinfo_unix_test.cpp

```diff
@@ -51,6 +51,13 @@ using std::string;
 #define safe_strcat(x, y) strlcat(x, y, sizeof(x))
 #define LINUX_LIKE_SYSTEM (defined(__linux__) || defined(__GNU__) || defined(__GLIBC__)) && !defined(__HAIKU__)
 
+#if WASM
+    #include <emscripten.h>
```

Comment:
Please move this include (in detailed described in the comment to the previous file)